cp3pt0-deployment/setup_tenant.sh

```diff
@@ -419,10 +419,10 @@ EOF
     # ${TETHERED_NS} is comma delimited, so need to replace commas with space
     local nss_exists="fail"
     if [ $PREVIEW_MODE -eq 0 ]; then
-        nss_exists=$(${OC} get nss common-service -n $OPERATOR_NS || echo "fail")
-    fi 
+        nss_exists=$(${OC} get nss common-service -n $OPERATOR_NS --ignore-not-found)
+    fi
 
-    if [[ $nss_exists != "fail" ]]; then
+    if [[ ! -z "$nss_exists" ]]; then
```

Comment:
Should we remove `local nss_exists='fail'`?

Otherwise, it will always assume NSS CR exists when the `preview_mode` is on.